eva-accession-import/src/main/java/uk/ac/ebi/eva/accession/dbsnp/processors/SubmittedVariantDeclusterProcessor.java

```diff
@@ -95,8 +94,7 @@ private boolean isSameType(DbsnpClusteredVariantEntity clusteredVariant, Submitt
         }
     }
 
-    private DbsnpSubmittedVariantEntity decluster(DbsnpSubmittedVariantEntity nonDeclusteredVariantEntity,
-                                                  SubmittedVariant variant,
+    public DbsnpSubmittedVariantEntity decluster(DbsnpSubmittedVariantEntity nonDeclusteredVariantEntity,
```

Comment:
I'm not totally happy with this visibility change, but I can't think of a simple way to do it either.